Jenkinsfile

```diff
@@ -57,13 +57,40 @@ node {
 
         sh "${dockerCompose} ${exec} zap-proxy zap-cli report -o /zap/reports/zap-scan-report.html -f html"
         archiveArtifacts 'reports/zap-scan-report.html'
+
+        stage('Checkout Legal') {
+          checkoutLegalIntegrationTests()
+        }
+
+        stage('Update images') {
+          sh "${legalDockerCompose} pull"
+        }
+
+        stage('Start & setup legal environment') {
+          sh "${legalDockerCompose} up -d zap-proxy remote-webdriver legal-frontend"
+          sh "./bin/set-legal-scanning-exclusions.sh ${execParams}"
+        }
+
+        stage('Run user journey through ZAP') {
```

Comment:
Can you please rename this and corresponding citizen frontend stages for clarity?